src/components/ProgressIndicator/MobileButton.js

```diff
@@ -1,35 +1,32 @@
 import PropTypes from 'prop-types';
+import {forwardRef} from 'react';
 
 import FAIcon from 'components/FAIcon';
-import {getBEMClassName} from 'utils';
 
-const MobileButton = ({
-  ariaMobileIconLabel,
-  accessibleToggleStepsLabel,
-  formTitle,
-  expanded,
-  onExpandClick,
-}) => {
-  return (
-    <button
-      className={getBEMClassName('progress-indicator__mobile-header')}
-      aria-pressed={expanded ? 'true' : 'false'}
-      onClick={onExpandClick}
-    >
-      <FAIcon
-        icon={expanded ? 'chevron-up' : 'chevron-down'}
-        modifiers={['normal']}
-        aria-label={ariaMobileIconLabel}
-      />
-      <span
-        className={getBEMClassName('progress-indicator__form-title')}
-        aria-label={accessibleToggleStepsLabel}
+const MobileButton = forwardRef(
+  ({ariaMobileIconLabel, accessibleToggleStepsLabel, formTitle, expanded, onExpandClick}, ref) => {
+    return (
+      <button
+        ref={ref}
+        className="openforms-progress-indicator__mobile-header"
+        aria-pressed={expanded ? 'true' : 'false'}
+        onClick={onExpandClick}
       >
-        {formTitle}
-      </span>
-    </button>
-  );
-};
+        <FAIcon
+          icon={expanded ? 'chevron-up' : 'chevron-down'}
+          modifiers={['normal']}
+          aria-label={ariaMobileIconLabel}
+        />
+        <span
+          className="openforms-progress-indicator__form-title"
+          aria-label={accessibleToggleStepsLabel}
+        >
+          {formTitle}
+        </span>
+      </button>
+    );
+  }
+);
 
 MobileButton.propTypes = {
```

Comment:
`ref` is not a prop :) it's a built in mechanism, that's also why `React.forwardRef` is used because otherwise the `ref` prop would get swallowed by the `MobileButton` component and not be available inside the props.